go/chat/utils/utils.go

```diff
@@ -1781,8 +1784,91 @@ func DecorateBody(ctx context.Context, body string, offset, length int, decorati
 	if err != nil {
 		return res, 0
 	}
-	strDecoration := fmt.Sprintf("%s%s%s", decorateBegin, string(out), decorateEnd)
+	//b64out := string(out)
```

Comment:
delete this?

Reply:
It was useful to toggle on/off for tests, so I kind of wanted it to just be easily accessible in the future if I needed it.

Reply:
sure, just checking.